### ISSUE TITLE
References to "Submission Redirect" label

### ISSUE THREAD

**Issue by horus68** [joomla/joomla-cms#17963]
### Steps to reproduce the issue
File: administrator/language/en-GB/en-GB.com_content.ini
the string COM_CONTENT_CREATE_ARTICLE_CUSTOM_CANCEL_REDIRECT_DESC points to a '**Submission Redirect**' label

```
COM_CONTENT_CREATE_ARTICLE_CUSTOM_CANCEL_REDIRECT_DESC="If set to 'Yes', you can set a redirection page, distinct from above 'Submission Redirect', to redirect to when user Cancels article submission.<br />If set to 'No', when user Cancels article submission he is redirected to the above 'Submission Redirect' page."
```
https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.com_content.ini#L35

but there is no "Submission Redirect" label

Actually its named "Submission/Cancel Redirect" from
```
COM_CONTENT_CREATE_ARTICLE_REDIRECTMENU_LABEL="Submission/Cancel Redirect"
```
https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.com_content.ini#L38

### Expected result
References to a label in description needs to be accurate

### System information (as much as possible)
Latest version on Github

**Comment by brianteeman** [comment]
Please submit a pull request https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests

**Comment by ghost** [comment]
closed as having Pull Request #18555
